gcsfs/core.py

```diff
@@ -455,23 +492,34 @@ async def _get_object(self, path):
         # Work around various permission settings. Prefer an object get (storage.objects.get), but
         # fall back to a bucket list + filter to object name (storage.objects.list).
         try:
-            res = await self._call("GET", "b/{}/o/{}", bucket, key, json_out=True)
+            print("GET", bucket, key, generation)
```

Comment:
Leftover debug - or should this be logging?

Reply:
This was leftover debugging, it's been removed now